src/components/Editor.tsx

```diff
@@ -1,6 +1,26 @@
 "use client";
-import MonacoEditor from "@monaco-editor/react";
+import MonacoEditor, { type Monaco } from "@monaco-editor/react";
+
+function configureEditor(monaco: Monaco) {
+  monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+    enableSchemaRequest: true,
+    validate: false,
```

Comment:
I think this is the behavior we actually want for now:

```suggestion
    validate: true,
    schemaValidation: "ignore",
```

https://microsoft.github.io/monaco-editor/docs.html#interfaces/languages.json.DiagnosticsOptions.html#schemaValidation

We could probably improve the actual error rendering, since I guess those schema issues should be considered `warning` by default... but I think it's best to just ignore them for now.

Reply:
I think that's okay for now given how early we are in the project here, but I might mention that the Flows def schema doesn't validate many of the currently deployed Flows to the Automate team